File: Desafios em C#/1893 - Fases da Lua.cs
using System; 

//Developed by: @LucasMarcuzo
class URI1893 {

    static void Main(string[] args) { 

        string[] linha = Console.ReadLine().Split(' ');
 
        int inicio = int.Parse(linha[0]);
        int fim = int.Parse(linha[1]);
        
        if(inicio >= 0 && fim <= 2 ) Console.WriteLine("nova");
        else if(inicio >= 3 && fim <= 96 && inicio > fim) Console.WriteLine("minguante");
        else if(fim >= 97 && inicio <= 100) Console.WriteLine("cheia");
        else if (fim >= 3 && inicio <= 96) Console.WriteLine("crescente");
        
        

    }

}

File: Desafios em JavaScript/1893 - Fases da Lua.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split(' ');

var inicio = parseInt(lines.shift());
var fim = parseInt(lines.shift());

if(inicio >= 0 && fim <= 2 ) 
{
    console.log("nova");
}
else if(inicio >= 3 && fim <= 96 && inicio > fim)
{ 
    console.log("minguante");
}
else if(fim >= 97 && inicio <= 100)
{
    console.log("cheia");
}
else if (fim >= 3 && inicio <= 96)
{
    console.log("crescente");
}